language/testing/Language_mock_gen.go

```diff
@@ -3,9 +3,8 @@
 package languagetesting
 
 import (
-	log "log"
-
 	mock "github.com/stretchr/testify/mock"
+	log "github.com/symflower/eval-dev-quality/log"
```

Comment:
Just wondering: how are we defining the log package? Because when you do "make generate", we overwrite it again?

Reply:
No. tried it.